src/scales/scale.time.js

```diff
@@ -35,10 +39,94 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'series',   // 'linear|series'
+			source: 'auto'    // 'auto|labels'
 		}
 	};
 
+	function buildLookupTable(ticks, min, max, linear) {
```

Comment:
This lookup table idea could probably be used as a better solution to handling 0 in the log axis. We would define a 0 point, and then the rest of it is a logarithmic interpolation region.